Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-drop.txt

MongoDB\\Collection::drop()
===========================

.. meta::

>    :description: Explore how to use the `MongoDB\Collection::drop()` method to remove a collection, including options for write concern and session management.

Reviewer: ```suggestion
   :description: Remove a collection with the MongoDB PHP Library, including options for write concern and session management.
```
